.github/guides/MAPS_AND_AWAY_MISSIONS.md

```diff
@@ -29,6 +29,23 @@ Instead, /tg/station map maintainers will always recommend using one of two mode
 
 Both of the above programs have native TGM support, which is mandatory for all maps being submitted to this repository. Anytime you want to make changes to a map, it is imperative you use the [Map Merging tools](https://tgstation13.org/wiki/Map_Merger). When you clone your repository onto your machine for mapping, it's always a great idea to run `tools/hooks/Install.bat` at the very start of your mapping endeavors, as this will install Git hooks that help you automatically resolve any merge conflicts that come up while mapping.
 
+## UPDATEPATHS
+
+#### Using UpdatePaths is mandatory for all mass-path changes in this repository.
```

Comment:
i think i was zealous writing this part let me know if i should change it

Reply:
Nah I think it should be left in, speaking from a mapper PoV. It's just easier to run the script(s) and have the maps be updated retrospectively as opposed to doing it by hand, which can be tedious and annoying.

Just my two cents.